src/styles/reviews.scss

```diff
@@ -0,0 +1,43 @@
+@import 'variables';
+
+.reviews {
+  display: flex;
+  box-sizing: border-box;
+  margin: 0 auto 24px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the order of CSS properties should be: position, box model, typography, visual, animation, misc. In this case, 'margin' should come after 'box-sizing'.